addons/binding/org.openhab.binding.velbus/README.md

```diff
@@ -0,0 +1,162 @@
+|  vmbgp4     |              |        |            |                X              |       X        |
+|  vmbgp4pir  |              |        |            |                X              |       X        |
+|  vmbgpo     |              |        |            |                X              |       X        |
+|  vmbgpod    |              |        |            |                X              |       X        |
+|  vmbpiro    |              |        |            |                X              |       X        |
+
+The type of a specific device can be found in the configuration section for things in the PaperUI. It is part of the unique thing id which could look like:
+
+```
+velbus:vmb4ryld:0424e5d2:01:CH1
+```
+
+The thing type is the second string behind the first colon and in this example it is **vmb4ryld**.
+
+## Discovery
+
+The Velbus bridge cannot be discovered automatically. It has to be added manually by defining the serial port of the Velbus Configuration module.
+
+Once the bridge has been added as a thing, a manual scan can be launched to discover all other supported Velbus devices on the bus. These devices will be available in the inbox. The discovery scan will also retrieve the channel names of the Velbus devices.
+
+## Thing Configuration
+
+The Velbus bridge needs to be added first in the things file or through Paper UI.
+It is necessary to specify the serial port device used for communication.
+On Linux systems, this will usually be either `/dev/ttyS0`, `/dev/ttyUSB0` or `/dev/ttyACM0` (or a higher  number than `0` if multiple devices are present).
+On Windows it will be `COM1`, `COM2`, etc.
+
+In the thing file, this looks e.g. like
+
+```
+Bridge velbus:bridge:1 [ port="COM1" ]
+```
+
+For the other Velbus devices, the thing configuration has the following syntax:
+
+```
+Thing velbus:<thing type>:<bridgeId>:<thingId> "Label" @ "Location"
+```
+
+or nested in the bridge configuration:
+
+```
+<thing type> <thingId> "Label" @ "Location"
+```
+
+The following thing types are valid for configuration:
+
+```
+vmb2ble, vmb2pbn, vmb4ryld, vmb4dc, vmb6pbn, vmb7in, vmb8pbu, vmbgp1, vmbgp2, vmbgp4, vmbgp4pir, vmbgpo, vmbgpod, vmbpiro
+```
+
+`thingId` is the hexadecimal Velbus address of the thing.
+
+`"Label"` is an optional label for the thing.
+
+`@ "Location"` is optional, and represents the location of the thing.
+
+
+## Channels
+
+For thing type `vmb2ble` the supported channels are `CH1` and `CH2`. UpDown, StopMove and Percent command types are supported.
+
+Thing types `vmb2pbn`, `vmb6pbn`, `vmb7in` and `vmb8pbu` have 8 trigger channels `CH1` ... `CH8`.
+
+For thing type `vmb4ryld` 5 channels are available `CH1` ... `CH5`.
+OnOff command types are supported.
+
+For thing type `vmb4dc` the supported channels are `CH1` ... `CH4`.
+OnOff and Percent command types are supported.
+Sending an ON command will switch the dimmer to the value stored when last turning the dimmer off.
```

Comment:
I think this is only correct if the last value resulted from a `PercentType` command via openHAB, right? So if a user changes the value externally, openHAB will not be notified about this change and thus it cannot restore the correct value, right?

Reply:
This is also the case when a user changes the value externally, because the last value is stored on the device, and restored with a "Restore Last Dim Value" command.

Reply:
The "Restore Last Dim Value" command is a feature that is supported by the device, and sent by the binding to implement the OpenHab "ON" command. Here is a screenshot of the relevant part of the device protocol:

![image](https://user-images.githubusercontent.com/12730069/38521464-11613456-3c46-11e8-9720-3d321fc396d8.png)


It is correct that the module stores its current dim level and uses this last state when it is power-cycled. The binding does not implement the logic to store the last dimvalue, but the implementation of the OpenHab ON command on this thing causes the dimmer to return to its last stored dim level. (See line 81 of the VelbusDimmerHandler class)

Reply:
Thank you for the clarification. This makes sense to me now.